### DISCUSSION TITLE
Colored navigation bar

### DISCUSSION THREAD

**Discussion by the8055u** [community/community#18709] [Mobile]
Implement a colored navigation bar in the GitHub mobile app to match the screen background please

**Comment by doguedogue** [comment]
You can change the theme in the settings section

![image](https://user-images.githubusercontent.com/23409026/173994397-e3da0e62-7e2e-41ee-a652-b8f5f6ae6ae7.png)

**Reply by the8055u** [reply]
but navigation bar is not transparent

**Comment by andsofine** [comment]
will this ever be fixed?
